freebsd/driver.h

```diff
@@ -0,0 +1 @@
+../linux/driver.h
```

Comment:
We need to figure out something better for this

Reply:
I don't understand. Is this about naming, licenses and such or about technical shortcomings?